app/views/layouts/application.html.erb

```diff
@@ -13,10 +13,15 @@
   </head>
 
   <body>
-    <div id="global-breadcrumb" class="header-context"></div>
     <div id="wrapper" role="main" class="<%= @presenter.content_class_dasherized %> guide">
+      <%= render partial: 'govuk_component/breadcrumbs', locals: { breadcrumbs: breadcrumbs } %>
```

Comment:
I think the breadcrumbs and related component should be in the `show` template rather than the application one.